### PR TITLE
Update recipe_sortable_listing.rst

### PR DESCRIPTION
**Pull request by vlepeule**
Fix a missing indentation level in the documentation (last YML code block in 6.3 section)

Closes #
Fixes #

## Changelog

```markdown
### Added
- Added some `Class::newMethod` to do great stuff

### Changed

### Deprecated

### Removed

### Fixed

### Security
```

## To do

- [ ] Update the tests
- [ ] Update the documentation
- [ ] Add an upgrade note

## Subject

Fix bad indentation

### CODE REVIEW

**Comment by vlepeule** [comment]
Just fix bad YML indentation

**Comment by greg0ire** [comment]
> I am targetting this branch, because…

ಠ_ಠ because what? Try to complete the sentence and you will see that you are targetting the wrong branch. Also, read the CONTRIBUTING.md to find out how to create a good commit message.

File: Resources/doc/cookbook/recipe_sortable_listing.rst [hunk 1]
```diff
@@ -193,8 +193,8 @@ And add the following call the ``admin.yml``
 	            - ~
 	            - AppBundle\Entity\Client
 	            - 'PixSortableBehaviorBundle:SortableAdmin'
```
Comment (greg0ire) [inline Resources/doc/cookbook/recipe_sortable_listing.rst]: Looks like there are tabs here (that's why it is red). Can you replace them?

File: Resources/doc/cookbook/recipe_sortable_listing.rst [hunk 2]
```diff
@@ -184,15 +184,15 @@ And add the following call the ``admin.yml``
...
+    services:
+        app.admin.client:
+            class: AppBundle\Admin\ClientAdmin
+            tags:
+                - { name: sonata.admin, manager_type: orm, label: "Clients" }
+            arguments:
+                - ~
+                - AppBundle\Entity\Client
+                - 'PixSortableBehaviorBundle:SortableAdmin'
                 calls:
```
Comment (greg0ire) [inline Resources/doc/cookbook/recipe_sortable_listing.rst]: Ok so now `calls` is no longer well indented :P

Comment (vlepeule) [inline Resources/doc/cookbook/recipe_sortable_listing.rst]: OMG sorry, I was struggling with Github editor...

**Comment by greg0ire** [comment]
Ok, it's great! Now please do the same, but target the right branch, fill in the template properly, and give a good commit message.

**Pull request closed without merging**